raddb/sites-available/coa-relay

```diff
@@ -123,24 +123,21 @@ server coa {
 		#  what these attributes are unfortunately, so we
 		#  cannot give more detailed advice here.
 		#
-		update control {
 
-			#
-			#  Example MySQL lookup
-			#
-#			&Tmp-String-0 := "%{sql:SELECT IFNULL(GROUP_CONCAT(CONCAT(nasipaddress,'#',acctsessionid) separator '|'),'') FROM (SELECT * FROM radacct WHERE ('%{User-Name}'='' OR UserName='%{User-Name}') AND ('%{Acct-Session-Id}'='' OR acctsessionid = '%{Acct-Session-Id}') AND AcctStopTime IS NULL) a}"
-
-			#
-			#  Example PostgreSQL lookup
-			#
-#			&Tmp-String-0 := "%{sql:SELECT STRING_AGG(CONCAT(nasipaddress,'#',acctsessionid),'|') FROM (SELECT * FROM radacct WHERE ('%{User-Name}'='' OR UserName='%{User-Name}') AND ('%{Acct-Session-Id}'='' OR acctsessionid = '%{Acct-Session-Id}') AND AcctStopTime IS NULL) a}"
+		#
+		#  Example MySQL lookup
+		#
+#		&control.Tmp-String-0 := %{sql:SELECT IFNULL(GROUP_CONCAT(CONCAT(nasipaddress,'#',acctsessionid) separator '|'),'') FROM (SELECT * FROM radacct WHERE ('%{User-Name}'='' OR UserName='%{User-Name}') AND ('%{Acct-Session-Id}'='' OR acctsessionid = '%{Acct-Session-Id}') AND AcctStopTime IS NULL) a}
```

Comment:
the `".."` quotes should be left here.  The parser still doesn't like `%{...}` with spaces in it.